ncolony/process_events.py

```diff
@@ -82,5 +91,10 @@ def message(self, contents):
         elif tp == 'RESTART-ALL':
             self.monitor.restartAll()
             log.msg("Restarting all monitored processes")
+        elif tp == 'RESTART-GROUP':
+            log.msg("Restarting group", contents['group'])
+            for name in self._groupToProcess[contents['group']]:
+                log.msg("Restarting monitored process: ", name)
+                self.monitor.stopProcess(name)
```

Comment:
simply stopping it will cause a restart?

Reply:
Yes, ProcessMonitor is a convergence loop 😄